src/pages/SingleCard/SingleCard.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { CardsContext } from '../../context/cardsContext';
+
+import styles from './singleCard.module.css';
```

Comment:
```suggestion
import s from './singleCard.module.css';
```
можно так. Обычно так общепринятно, чтобы не писать длинное `styles`